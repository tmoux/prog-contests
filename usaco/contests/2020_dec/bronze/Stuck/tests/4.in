35
N 25 0
E 28 57
N 7 62
N 36 11
E 6 41
N 69 34
N 40 59
N 67 17
N 31 5
N 47 67
E 39 54
E 38 38
N 10 9
E 53 18
N 11 35
N 8 44
E 41 10
N 44 4
E 17 36
N 56 2
N 65 37
E 1 28
N 15 23
E 68 33
N 18 29
E 34 53
N 13 58
E 63 20
N 0 69
E 20 25
N 64 56
N 12 32
E 35 1
N 58 14
N 3 51
