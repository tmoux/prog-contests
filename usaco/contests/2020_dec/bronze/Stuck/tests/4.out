25
36
Infinity
25
59
Infinity
Infinity
Infinity
20
Infinity
26
27
19
5
6
Infinity
15
6
41
16
Infinity
14
18
Infinity
7
31
Infinity
4
Infinity
16
Infinity
9
Infinity
24
Infinity
